src/pages/components/toggle/usage.mdx

```diff
@@ -67,76 +78,130 @@ user “flips the switch”. They are commonly used for “on/off” switches.
 </Column>
 </Row>
 
-<Caption>Default and small toggle variants</Caption>
+<Caption>Default and small toggle variants.</Caption>
+
+## Formatting
+
+### Anatomy
+
+Default and small toggle both comprise of a component label, the toggle button
+and an action text. Default toggles are required to display component label and
+action text unlike the small toggle where both these elements are optional.
+
+<Row>
+<Column colLg={8}>
+
+![Anatomy of Default and small toggle variants](images/toggle-usage-anatomy.png)
+
+</Column>
+</Row>
+
+<Caption>Anatomy of Default and Small toggle.</Caption>
+
+<Row>
+<Column colSm={2} colMd={4} colLg={4}>
+
+#### 1. Default toggle
+
+A. Label <br /> B. Toggle <br /> C. Action text <br />
+
+</Column>
+<Column colSm={2} colMd={4} colLg={4}>
+
+#### 2. Small toggle
+
+A. Label (optional) <br /> B. Toggle <br /> C. Action text (optional) <br />
+
+</Column>
+</Row>
 
 ## Content
 
 #### Label text
 
 Label text must accompany a toggle to further clarify the action that the toggle
-performs.
+performs. Label text is optional for Small toggle.
 
 #### Action text
 
 Use text to describe the binary action of toggle so that the action is clear.
 Action text must be three words or less and is displayed on the side of a
-toggle.
+toggle. Action text is optional for Small toggle.
 
 #### Language
 
 Use adjectives rather than verbs to describe actions and the state of the object
 affected.
 
+### Further guidance
+
+For further content guidance, see Carbon’s
+[content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
+
 ## Behaviors
 
 ### States
 
-Toggles have six states: on, off, focus, disabled, skeleton, and read-only.
+The Default and Small toggles have six states: on, off, focus, disabled,
+read-only and skeleton. For more information on toggle states, see the
+[style tab](https://carbondesignsystem.com/components/toggle/style/).
+
+| State     | When to use                                                                                                                                                                                                                                                 |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| On        | When a user clicks or uses the arrows keys to switch on the component.                                                                                                                                                                                      |
```

Comment:
According to the [Usage tab template](https://arc.net/l/quote/nrcrdgke), if the 'State' column does contain links, it should be displayed in italics.

<img width="725" alt="Screenshot 2024-08-26 at 10 16 34 PM" src="https://github.com/user-attachments/assets/ba7a8be8-a7e4-421b-a191-7a034260c7d5">

Reply:
I know it says that in our template right now Kritvi, but I think we will be moving away from this according to the guidance we got recently from Tom and Aubrey here: https://ibm.ent.box.com/notes/1530339018364.

Reply:
Oh got it Lauren!